doc/lua_api.txt

```diff
@@ -5073,8 +5073,8 @@ Call these functions only at load time!
 * `minetest.register_on_punchnode(function(pos, node, puncher, pointed_thing))`
     * Called when a node is punched
 * `minetest.register_on_generated(function(minp, maxp, blockseed))`
-    * Called after generating a piece of world. Modifying nodes inside the area
-      is a bit faster than usually.
+    * Called after generating a piece of world. Modifying nodes inside this area
```

Comment:
"this" refers to something previously mentioned, but there's no area previously mentioned. only the function definition.

Reply:
Pardon, I felt as if 'this' referenced the area represented by minp and maxp. I will revert this change.